### ISSUE TITLE
Why not give Koibisto MPV option

### ISSUE THREAD

**Issue by shenjian333333** [Luecx/Koivisto#232]
In some chessGUI , the engine must have MPV option to analysis games . （Like lucas chess）

**Comment by tissatussa** [comment]
> In some chessGUI, the engine must have `MPV` option to analysis games. (Like lucas chess)

@shenjian333333 no, an engine "must" NOT have MPV to analyse games, even in Lucas Chess, it's an option .. see `Mass analysis` https://chessionate.com/lucaswiki/index.php?title=Mass_analysis : here `MPV` is called `MultiPV`.

in general it's stated that `MPV` 1 is best for playing, but Koivisto DOES have `MPV` as one of its `UCI` options, so this can be used to analyse games - i guess you were mistaken .. Indeed, using `MPV` can be nice to see different lines when analysing a position, and Koivisto can do it.

**Comment by Luecx** [comment]
We added MPV at some point but removed it again. The UCI option is an old relict. We had a very crucial bug when using MPV = 1 but using SMP. So we removed it and may add it again in the future

**Comment by shenjian333333** [comment]
> > 在某些 chessGUI 中，引擎必须具有`MPV`分析游戏的选项。（就像卢卡斯国际象棋一样）
> 
> @shenjian333333不，引擎“必须”没有 MPV 来分析游戏，即使在 Lucas Chess 中，它也是一个选项。请参阅`Mass analysis` https://chessionate.com/lucaswiki/index.php?title=Mass_analysis：这里`MPV`称为`MultiPV`。
> 
> 一般来说，它表示`MPV`1 最适合玩，但 Koivisto 确实有`MPV`它的选项之一`UCI`，所以这可以用来分析游戏 - 我猜你错了..确实，`MPV`在分析 a 时使用可以很好地看到不同的线条位置，科伊维斯托可以做到。

But I can`t use alnalysis with koibisto in lucas chess , even set mpv=1
